### ISSUE TITLE
Support nos script in .NET Core builds

### ISSUE THREAD

**Issue by nasser** [nasser/nostrand#26]
We should provide a `nos` script for the .NET Core builds

**Comment by skydread1** [comment]
bin/x64/Debug/netcoreapp3.0/nos
```
#!/bin/bash
ACTUAL_PATH=`dirname "$0"`
LINKED_PATH=`readlink "$0"`
ASSEMBLY_PATH=$([ "$LINKED_PATH" ] && echo `dirname "$LINKED_PATH"` || echo "$ACTUAL_PATH")
dotnet $ASSEMBLY_PATH/Nostrand.dll "$@"
```

**Comment by nasser** [comment]
closed by #30, thanks @skydread1!
